_includes/release-notes/15-1.html

```diff
@@ -0,0 +1,52 @@
+<div class="component-slim grid-wrapper">
+  <div class="width-12-12">
+    <h4>Infinispan 15.1 Release Notes</h4>
```

Comment:
15.2

Reply:
You are looking at 15.1 release notes, which I forgot to add last time

Reply:
oh, makes sense sorry